CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Added
+- Add `verify-snaps` script to verify snaps in the registry ([#67](https://github.com/MetaMask/snaps-registry/pull/67)) ([#80](https://github.com/MetaMask/snaps-registry/pull/80))
```

Comment:
This is not part of the published package, so I don't think you need to mention this.